internal/docstore/firedocstore/fs.go

```diff
@@ -64,7 +64,25 @@ func newCollection(client *vkit.Client, projectID, collPath, nameField string) *
 
 // RunActions implements driver.RunActions.
 func (c *collection) RunActions(ctx context.Context, actions []*driver.Action) (int, error) {
-	groups := groupActions(actions)
+	// Split the actions into groups, each of which can be done with a single RPC.
+	// - Consecutive writes are grouped together.
+	// - Consecutive gets with the same field paths are grouped together.
+	// TODO(jba): when we have transforms, apply the constraint that at most one transform per document
+	// is allowed in a given request (see write.proto).
+	groups := driver.SplitActions(actions, func(cur, new *driver.Action) bool {
```

Comment:
Consider pull the func out to its standalone function since it doesn't use `actions` inside and there are a lot of documentations with it.

Reply:
Done.